server/functions/format_type.go

```diff
@@ -46,4 +46,5 @@ var format_type = framework.Function2{
 		}
 		return "???", nil
 	},
+	Strict: false,
```

Comment:
Mentioned earlier, but we don't need to specify that `Strict` is `false` since it's the default value.